### PR TITLE
Update defining-default-values-with-the-shouldserialize-and-reset-methods.md

### PR DESCRIPTION
**Pull request by RussKie**
## Summary

Make the methods private as those are invoked by convention.

### CODE REVIEW

File: docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md
Comment (Youssef1313) [inline docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md:36]:
```suggestion
private void ResetMyFont()
{
```

File: docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md
Comment (Youssef1313) [inline docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md:50]:
```suggestion
   Return thefont IsNot Nothing
```

File: docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md
Comment (Youssef1313) [inline docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md:57]:
```suggestion
private bool ShouldSerializeMyFont()
{
```

File: docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md
Comment (Youssef1313) [inline docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md:97]:
```suggestion
      Return thefont IsNot Nothing
```

File: docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md
Comment (Youssef1313) [inline docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md:130]:
```suggestion
   private bool ShouldSerializeMyFont()
   {
```

File: docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md
Comment (Youssef1313) [inline docs/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md:134]:
```suggestion
   private void ResetMyFont()
   {
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
nits

These suggestions can also be applied to other parts of the article.

**Comment by BillWagner** [comment]
@RussKie Can you apply the suggestions above? Then, this is ready to merge.

**Comment by RussKie** [comment]
@BillWagner I can't merge without an approval. 
Squash-merge is recommended.

**Review by BillWagner — APPROVED** [review]
This LGTM @RussKie 

Once you resolve the merge conflicts, this is ready to :shipit:

**Comment by adegeo** [comment]
@RussKie This content was moved to https://github.com/dotnet/docs-desktop and you'll have to file the PR there. Sorry about that! Bad timing!

@BillWagner fyi, can't fix the merge conflict because the file doesn't exist 😄

**Comment by RussKie** [comment]
> fyi, can't fix the merge conflict because the file doesn't exist 😄

BOO! 😈 
Here's the new one - https://github.com/dotnet/docs-desktop/pull/34

**Pull request closed without merging**
